framework/include/auxkernels/TagVectorAux.h

```diff
@@ -26,6 +25,7 @@ class TagVectorAux : public TagAuxBase<AuxKernel>
 protected:
   virtual Real computeValue() override;
 
+  const bool _unscaled;
```

Comment:
Probably good to add a short doxygen comment describing clearly what this is since it can be ambiguous.